content/en/sidecar-container.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sidecar Container
+status: Feedback Appreciated
+category: concept
+---
+
+The Sidecar Container is an implementation of the sidecar pattern. 
+An application deployed on a second container will run side by side and share its lifecycle with a primary application running on a main container.
+
+## Problem it addresses
+
+There are scenarios where it is convenient to group [containers](/container/) to deal with problems related to the need to support cross-platform requirements or shared functionality. 
+Problems like logging, monitoring, tracing or [observability](/observability/) in general, and security, traffic management, networking and common configurations applicable to multiple applications. These shared functionalities can be implemented independently as services and run on separate containers.
+
+## How it helps
+
+The term "sidecar" comes from the representation of a classic motorcycle with its attached sidecar. In this metaphor, the bike represents the main container and the sidecar the secondary container. 
+This heterogeneous approach consists of using a separated and isolated process running on a secondary container to address problems that aren't unique to a single application but rather to the entire platform where the application is running. 
```

Comment:
```suggestion
This approach consists of using a separated process running on a secondary container to address problems that aren't unique to a single application but rather to the entire platform where the application is running. 
```

Rathing than just saying that it solves problems that are not unique, we should name those problems. Like a sidecar shares same network and storage with the main container. That is I guess the main advantage, isn't it?

Reply:
@iamNoah1 accepted your review and recommendations in commit 5173e2a985105d8e0f8b518b4be368f9ff699f49